azure/key_vault/terraform.tf

```diff
@@ -13,22 +13,3 @@ terraform {
 
   required_version = ">= 1.3.0, < 2.0.0"
 }
-
-provider "azurerm" {
-  features {
-    key_vault {
-      purge_soft_delete_on_destroy               = true
-      purge_soft_deleted_certificates_on_destroy = true
-      purge_soft_deleted_keys_on_destroy         = true
-      purge_soft_deleted_secrets_on_destroy      = true
-
-      # When recovering soft-deleted Key Vault items (Keys, Certificates, and Secrets)
-      # the Principal used by Terraform needs the "recover" permission.
-      recover_soft_deleted_key_vaults   = true
-      recover_soft_deleted_certificates = true
-      recover_soft_deleted_keys         = true
-      recover_soft_deleted_secrets      = true
```

Comment:
These features need to be passed on to the Service, correct?

Reply:
correct